pkg/analytics/instrumentation_collector.go

```diff
@@ -228,13 +232,14 @@ func toInteractionErrors(errors []error) *[]api.InteractionError {
 
 func toInteractionError(e error) *api.InteractionError {
```

Comment:
Could we add a test case for this function?

Reply:
Done!